### ISSUE TITLE
Time difference between frames is not constant

### ISSUE THREAD

**Issue by eunsanMountain** [argoai/argoverse-api#124]
![image](https://user-images.githubusercontent.com/46332431/80555924-c8b88d00-8a0c-11ea-9edf-26b75747c818.png)

First of all, thank you very much for a large amount of processed data. There are no problems for research purposes, but there are some questions about the dataset in competition.

In test dataset 2, 13, 75, etc., we can see that the time difference between frames is not constant. In the paper, the time difference between frames of the dataset is 10hz, but the difference between the actual dataset is 6hz or less.

I think noise from the observation dataset is helpful for robust learning, but I think the exact time interval(10Hz) is important to compare the predicted trajectory and ground truths. (The error between generating 30 frames of trajectory at 10 Hz and generating 6 Hz is quite large)

I think it is impossible to predict the frame's interval of data when t> 2s of the test dataset. To solve this, what is the time difference between frames in the test dataset when t> 2s? Is it fixed or can you provide it?

**Comment by James-Hays** [comment]
You're right that the sequences don't always have a 10hz (100ms per tick) sample rate. For the next update to Argoverse forecasting data (which won't happen during this current competition), we will try to address this problem. 

I imagine most forecasting methods using Argoverse make the assumption that the tick rate is constant (i.e. no jitter https://en.wikipedia.org/wiki/Jitter ) and probably don't even look at the timestamps. Within each sequence, jitter is mostly low. If we measure jitter as the standard deviation among tick durations in milliseconds, and look at the full 5 second sequences in the validation set, the 5% of sequences with the worst jitter have a standard deviation of 17.6 ms and the 1% of sequences with the worst jitter have a standard deviation of 41.4 ms.

Beyond just analyzing jitter, it's of course not desirable for a sequence to have low jitter but a tick duration that isn't 100 ms, e.g. if the ticks are uniformly 120 ms. But if a sequence has a uniformly longer tick duration, I don't think that will break most forecasting methods. It will appear that everything in the scene is moving faster if the model has implicitly learned the 100 ms tick duration, and that will lead to reasonable forecasts in most cases.

**Comment by zhaone** [comment]
Hi, thank you for providing such a valuable dataset. I am confused about this issue recently after did some statistical work about the val dataset, see [my jupyter notebook](https://render.githubusercontent.com/view/ipynb?commit=f6968ed28fdf58bb845bbf5bf2c0668c8cf19ea1&enc_url=68747470733a2f2f7261772e67697468756275736572636f6e74656e742e636f6d2f7a68616f6e652f6172676f76657273652d6170692f663639363865643238666466353862623834356262663562663263303636386338636631396561312f6d7953706163652f6578706c6f72654172676f4170692e6970796e62&nwo=zhaone%2Fargoverse-api&path=mySpace%2FexploreArgoApi.ipynb&repository_id=297655148&repository_type=Repository#time-stamp). Briefly speaking, according to the quantile fig of delta time ![quantile](https://github.com/zhaone/argoverse-api/blob/zhaoyi/mySpace/val_time_stamp_quantile.png), the sampling frame rate fluctuates to some extent. There are about 17% samples are lower than 9HZ and the biggest delta time between 2 frames is 2.5s.

The key point I want to confirm is **whether frame rate of the test ground truth is strictly 10 HZ when you evaluate my algorithm**, since I could interpolate the frame rate in train and evaluation sets to 10HZ, but for the test set, without the timestamp, I am not able to do interpolation.

**Comment by James-Hays** [comment]
Hi again. Your analysis is great. Thanks for that.

The test set is geographically distinct from the train and validation sets, but it has the same generation procedure and the same evaluation metrics as you would use on the validation set. The distribution of tick durations should be similar.

**Comment by EadMan46** [comment]
I totally agree with @zhaone .
I also interpolated during training and assume 10Hz in the test sequence. 
When checking my validation results: the MissRate dramatically increases when ignoring the label's timestamps.
I recommend providing the future timestamps of the target agent in the test set (which you have in your test evaluation data).
This will allow for a more proper evaluation, that shows which algorithm would work better in real time.

**Comment by James-Hays** [comment]
Hi @EadMan46 .

Argoverse 2.0 will address this.
